### ISSUE TITLE
Build error manifest parse error

### ISSUE THREAD

**Issue by revolter** [nicklockwood/SwiftFormat#590]
After following [these steps](https://github.com/nicklockwood/SwiftFormat#xcode-build-phase), I get this error when building the target:

```
/Users/revolt/Development/Project/BuildTools: error: manifest parse error(s):
<unknown>:0: warning: using sysroot for 'iPhoneSimulator' but targeting 'MacOSX'
<unknown>:0: error: module 'Swift' was created for incompatible target x86_64-apple-ios13.0: /var/folders/88/6r327zlj6bqf5t_xfzm495h00000gn/C/org.llvm.clang.revolt/ModuleCache/Swift-3IRLIPY6Y8BSL.swiftmodule
```

And now I'm getting:

```
/Users/revolt/Development/Project/BuildTools: error: manifest parse error(s):
<unknown>:0: warning: using sysroot for 'iPhoneOS' but targeting 'MacOSX'
<unknown>:0: error: unable to load standard library for target 'x86_64-apple-macosx10.10'
```

**Comment by nicklockwood** [comment]
Could this be related to https://github.com/nicklockwood/SwiftFormat/pull/586 ?

**Comment by revolter** [comment]
Oh, wow, I didn't think it could be caused by Xcode 11.4. I'll try it tonight.

**Comment by revolter** [comment]
Yes, it's fixed by #586. Thank you!
